Assets/MixedRealityToolkit/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Compilation;
+using UnityEditorInternal;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.MSBuild
+{
+    /// <summary>
+    /// A helper class to parse the state of the current Unity project.
+    /// </summary>
+    public class UnityProjectInfo
+    {
+        private const string SDKProjectTypeGuid = "FAE04EC0-301F-11D3-BF4B-00C04F79EFBC";
```

Comment:
>FAE04EC0 [](start = 51, length = 8)

Is this just an arbitrarily generated thing, or is there some secret or guidance for how we might want to rev or change it going forward?